lib/src/test/java/com/auth0/android/jwt/ClaimImplTest.java

```diff
@@ -241,4 +241,27 @@ public void shouldThrowIfListClassMismatch() throws Exception {
         exception.expect(DecodeException.class);
         claim.asList(UserPojo.class);
     }
+
+    @Test
+    public void shouldGetSubClaimsObject() throws Exception {
+        String username = "Sabin";
+
+        SubClaimsPojo data = new SubClaimsPojo();
+        data.setUsername(username);
+
+        JsonElement value = gson.toJsonTree(data);
+
+        ClaimImpl claim = new ClaimImpl(value);
+        SubClaimsPojo object = claim.asObject(SubClaimsPojo.class);
+
+        assertThat(object.getUsername(), is(username));
+    }
+
+    @Test
+    public void shouldGetNullIfSubClaimsIsNotAnObject() throws Exception {
```

Comment:
This may need to be changed if we agree on the first comment I left

Reply:
As I removed `value.isJsonObject()` check, this test case was no longer necessary.